recipes/ftgl/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "ftgl" %}
+{% set version = "2.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://github.com/frankheckenbach/ftgl/archive/v2.4.0.tar.gz"
+  sha256: "aa97da1c3442a8fd3941037655df18016d70b5266381c81d81e8b5335f196ea8"
+  patches:
+    - cmake_fix_2.4.1.patch
+
+build:
+  number: 0
+  run_exports:
+    # Hasn't been updated in years so pin tightly
+    - {{ pin_subpackage('ftgl', max_pin='x.x.x') }}
+  skip: true  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make  # [unix]
+    - ninja  # [win]
+    - cmake
+    # Required for OpenGL support
+    # See https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=cdt#libgl
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+  host:
+    - xorg-libxfixes  # [linux]
+    - libglu  # [linux]
+    - freetype
+    - libglu  # [linux]
+
+  run:
+    - libglu  # [linux]
+    - freetype
+    - libglu  # [linux]
```

Comment:
```suggestion
```

Reply:
This is needed to add `GL/glu.h`

Reply:
(I can remove it and retry to verify, but something was failing until I added this, either the CMake search or GL/glu.h)

Reply:
Oh, you mean it's (actually) duplicated

Reply:
Bad merge when rebasing.